Scripts/Core/VRView.cs

```diff
@@ -132,9 +133,8 @@ public void OnEnable()
 			m_Camera.farClipPlane = 1000f;
 
 			// Generally, we want to be at a standing height, so default to that
-			const float kHeadHeight = 1.7f;
 			Vector3 position = m_CameraRig.position;
-			position.y = kHeadHeight;
+			position.y = HeadHeight;
```

Comment:
I want to say this used to be done only for Oculus, but I can't quite remember. All I'm doing in this case is exposing the const so I can reset to the same position.